pdc/apps/utils/messaging.py

```diff
@@ -14,11 +14,9 @@
 
 
 class DummyMessenger(object):
-    def __init__(self):
-        pass
 
     def send_message(self, topic, msg):
-        pass
+        logger.info('Sending to %s:\n%s' % (topic, json.dumps(msg, sort_keys=True, indent=2)))
```

Comment:
Isn't `msg` already string?

Reply:
Right, it is. That will have to change probably.